### DISCUSSION TITLE
Adding Youtube as a Source

### DISCUSSION THREAD

**Discussion by RobertCajun** [SamTV12345/PodFetch#153] [Ideas]
Not super high priority I listen to the WAN Show on my commutes and it goes Live on YouTube on Fridays and then hits the Podcast Apps on Monday Afternoon.

Anyway to add the feature to monitor a YouTube Playlist and download the Audio (or video I guess) of new videos once they're added?

**Comment by MI3Guy** [comment]


**Reply by SamTV12345** [reply]
I think his problem is that he wants to listen to the podcast rightaway. For that something like a YouTube API key is necessary to access their channel history. But it's not yet implemented. I just took a quick look at it.
